contracts/LoanCore.sol

```diff
@@ -767,30 +764,31 @@ contract LoanCore is
      *      Transfers should occur in the calling function.
      *
      * @param loanId                 The ID of the loan to repay.
-     * @param _amountFromPayer       The amount of tokens to be collected from the repayer.
      * @param _amountToLender        The amount of tokens to be distributed to the lender (net after fees).
      * @param _interestAmount        The amount of interest to be paid.
      * @param _paymentToPrincipal    The amount of principal to be paid.
      *
      * @return data                  The loan data for the repay operation.
-     * @return loanRepaid            Bool denoting whether the loan is repaid.
      */
     function _handleRepay(
         uint256 loanId,
-        uint256 _amountFromPayer,
         uint256 _amountToLender,
         uint256 _interestAmount,
         uint256 _paymentToPrincipal
-    ) internal returns (LoanLibrary.LoanData memory data, bool loanRepaid) {
+    ) internal returns (LoanLibrary.LoanData memory data, uint256 totalRepayment) {
         data = loans[loanId];
         // Ensure valid initial loan state when repaying loan
         if (data.state != LoanLibrary.LoanState.Active) revert LC_InvalidState(data.state);
 
+        totalRepayment = _paymentToPrincipal + _interestAmount;
```

Comment:
Since the sum of `_paymentToPrincipal + _interestAmount` is used in a few places in the `_handleRepay` fn and also the `repay` and `forceRepay` fn's, I thought it was cleaner to do the math here and reuse the result in those other places.

Reply:
But you already get this same value on line 778, right?